### ISSUE TITLE
Because flutter_clock requires SDK version >=1.20.1 <2.0.0, version solving failed.

### ISSUE THREAD

**Issue by gionchat** [killermonk/lib_octal_clock.dart#1]
Error info：
![image](https://user-images.githubusercontent.com/58280808/104868995-cbcf2100-597f-11eb-87d3-785bcbb2df15.png)

**Comment by killermonk** [comment]
Yup... It's 3 years old and has not been updated to be compatible with Dart 2+.
Would you like to contribute?
